R/parse-remotes.R

```diff
@@ -37,7 +37,7 @@ github_rx <- function() {
     ## Optional remote type
     "(?:github::)?",
     ## Username
-    "(?<username>(?:[^/]+))/",
+    "(?<username>(?:[[:alnum:]-]+))/",
```

Comment:
This was needed so that the `github_rx()` did not match ssh Github URLs.

Reply:
Hmmm. Do you know what characters are allowed in usernames?

Reply:
From https://www.npmjs.com/package/github-username-regex

- Github username may only contain alphanumeric characters or hyphens.
- Github username cannot have multiple consecutive hyphens.
- Github username cannot begin or end with a hyphen.
- Maximum is 39 characters

Repositories seem to have similar rules.